src/common/appmain.cpp

```diff
@@ -62,10 +62,10 @@ void app_idle(void) {
 void app_run(void) {
     DBG("app_run");
 
-#ifdef ETHERNET
+#ifdef BUDDY_ENABLE_ETHERNET
     if (diag_fastboot)
         osThreadResume(webServerTaskHandle);
```

Comment:
i dont understand why it is like this? could you confirm with Robert what is this meant for?